plaso/parsers/plist_plugins/ios_carplay.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+"""Plist parser plugin for Car Play App plist files.
+   It contains history of opened apps on Car Play App.
+"""
+
+from dfdatetime import posix_time
+
+from plaso.containers import plist_event
+from plaso.containers import time_events
+from plaso.lib import definitions
+from plaso.parsers import plist
+from plaso.parsers.plist_plugins import interface
+
+
+class IOSCarPlayPlugin(interface.PlistPlugin):
+  """Plist parser plugin for IOS Car Play App plist files.
+
+  """
+
+  NAME = 'ios_carplay'
+  DATA_TYPE = 'ios:carplay:history'
```

Comment:
Plugin does not need to define a data type, this should be done in an event data object